cloudigrade/instance/management/commands/check_instances.py

```diff
@@ -0,0 +1,28 @@
+"""Management command for checking currently-running AWS EC2 instances."""
+import logging
+
+from django.core.management.base import BaseCommand
+from django.utils.translation import ugettext as _
+
+from util import aws
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Command to display a list of running instances for a given ARN."""
+
+    help = _('Display a list of running instances for a given ARN.')
+
+    def add_arguments(self, parser):
+        """Add an --arn argument for use by this command."""
+        parser.add_argument(
+            '--arn', dest='arn', required=True,
```

Comment:
Good point. I was trying to think if there would be a case where this would expand to have more arguments in the future, but I don't have any now and it's easy enough to change later if needed. I'll simplify this down.